src/Auth0.Core/ApiError.cs

```diff
@@ -27,10 +28,23 @@ public class ApiError
         [JsonProperty("message")]
         public string Message { get; set; }
 
-        /// <summary>
-        /// The HTTP Status code.
-        /// </summary>
-        [JsonProperty("statusCode")]
-        public int StatusCode { get; set; }
+        public static ApiError ParseOrDefault(string responseContent)
+        {
+            if (String.IsNullOrEmpty(responseContent))
+                return default;
```

Comment:
**Question:** what does `return default` do?

Reply:
It's the same as `return null` for reference types and `return new x` for value types.  Given ApiError is a reference type I could just put `return null` here - I briefly toyed with making ApiError a struct hence the default here which works for both.